src/js/plugin-bridge/PluginModules.js

```diff
@@ -71,9 +71,8 @@ module.exports = {
     NestedServiceLinks: 'NestedServiceLinks',
     Page: 'Page',
     PageHeader: 'PageHeader',
-    ProgressBar: 'charts/ProgressBar',
     RequestErrorMsg: 'RequestErrorMsg',
-    RouterUtil: 'RouterUtil',
```

Comment:
This was listed under components (where it is not), but also under utils (as it should)